packages/desktop/electron/lib/menu.js

```diff
@@ -1,5 +1,5 @@
 import { app, ipcMain, Menu, shell } from 'electron'
-import { WalletRoutes, ExternalRoute } from 'shared/lib/typings/routes'
+import { WalletRoutes, ExternalRoute } from '@core/router/enum/routes'
```

Comment:
Just FYI, the modules should contain the barrel files so that this line can be either what it is now _or_ just `import { ... } from '@core/router'`. I think we should try and import code from the highest-level directory possible and only specify sub folders of a module when we absolutely have to, like within a module itself that needs to reference its own types or just to avoid circular imports.

Reply:
I suppose the case here is that we _can_ specify a specific location as that's all that's being used here.